app-utils/pmbootstrap/autobuild/defines

```diff
@@ -0,0 +1,4 @@
+PKGNAME=pmbootstrap
+ PKGSEC=utils
+ PKGDEP="python-3 openssl tar git util-linux procps" 
+ PKGDES="A cli tool for developing and installing PostmarketOS" 
```

Comment:
You should add `ABHOST=noarch` below since this is an architecture-agnostic package.

Reply:
For PKGDES:

Command line tool to install a postmarketOS system environment